project.clj

```diff
@@ -14,7 +14,8 @@
 
   :profiles {:dev {:plugins [[lein-cljsbuild "1.1.7"]
                              [lein-doo "0.1.11"]]
-                   :dependencies [[org.clojure/clojurescript "1.10.520"]]}}
+                   :dependencies [[org.clojure/clojurescript "1.10.520"]
+                                  [org.clojure/core.rrb-vector "0.1.1"]]}}
```

Comment:
why is that new dependency added?

Reply:
A conflicting version of rrb-vector was being pulled in transitively by another dep.

Reply:
originally we got this transitively but that version has a bug the blocks us from running CLJS tests. I pinned it here to fix this